cypress/e2e/tests/pages/manager/cluster-manager.spec.ts

```diff
@@ -111,7 +114,10 @@ describe('Cluster Manager', { testIsolation: 'off', tags: ['@manager', '@adminUs
     // re-enable the AKS kontainer driver
     KontainerDriversPagePo.navTo();
     driversPage.waitForPage();
-    driversPage.list().actionMenu('Azure AKS').getMenuItem('Activate').click();
+    driversPage.list().actionMenu('Azure AKS').getMenuItem('Activate').click()
+      .then(() => {
+        reenableAKS = false;
```

Comment:
Please, add an API call to ensure that the state of the container driver has successfully changed with the click() before changing `reenableAKS`